src/LumiPhotonChamber_geo.cpp

```diff
@@ -20,7 +20,9 @@ static Ref_t create_detector(Detector& description, xml_h e, SensitiveDetector /
   xml_comp_t    x_pos           = x_det.child( _Unicode(position) );
   //
   string        det_name        = x_det.nameStr();
-  string        mat_pipeCap     = getAttrOrDefault<string>( x_det, _Unicode(pipeAndCapMaterial), "Aluminum" );
+  string        mat_pipe        = getAttrOrDefault<string>( x_det, _Unicode(pipeMaterial), "Aluminum" );
+  string        mat_entrCap     = getAttrOrDefault<string>( x_det, _Unicode(entrCapMaterial), "Aluminum" );
+  string        mat_exitCap     = getAttrOrDefault<string>( x_det, _Unicode(exitCapMaterial), "Aluminum" );
```

Comment:
I suppose, it would not hurt to keep the default identical to xml spec.
```suggestion
  string        mat_exitCap     = getAttrOrDefault<string>( x_det, _Unicode(exitCapMaterial), "Beryllium" );
```